### PR TITLE
WIP: Preparing release

### PR DESCRIPTION
**Pull request by bbrala**
Preparing release

- [x] Create a proper README.
- [x] Update namespace.
- [x] Extract fixtures client.
- [x] Remove resources? Seems redundant.
- [x] Split service provider binds.
- [ ] Add changelog file.
- [ ] Perhaps increase coverage a bit.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,8 +1,295 @@
...
-# WORK IN PROGRESS
+A PHP package for mapping remote [{json:api}](http://jsonapi.org/) resources to Eloquent like models and collections.
 
-A PHP package for mapping remote {json:api} resources to Eloquent like models and Collections.
\ No newline at end of file
+
+## Installation
+
+``` bash
+composer require swisnl/jsonapi
```
Comment (bbrala) [inline README.MD]: Should this perhaps include a http adapter?

File: README.MD [hunk 2]
```diff
@@ -1,8 +1,295 @@
...
+  }
+},
+```
+
+
+## Todo
+
+ - [ ] CHANGELOG.MD
+ - [ ] Increase test coverage
+
```
Comment (bbrala) [inline README.MD]: This should be removed here imo and i'll add it to the PR comment.

File: README.MD [hunk 3]
```diff
@@ -1,8 +1,295 @@
...
+
+## Testing
```
Comment (bbrala) [inline README.MD]: Add a few lines about the FixturesClient here.

File: README.MD (hunk 1 above)
Comment (JaZo) [reply README.MD]: I've added a separate section about the HTTP client just under this section.

**Comment by JaZo** [comment]
@bbrala, i've addressed your comments!

**Pull request merged**